static/src/stylesheets/module/commercial/_labs-capi.scss

```diff
@@ -378,6 +378,13 @@
         color: $neutral-1;
     }
 
+    .fc-item__link,
+    .fc-sublink__link {
+        &:visited {
+            color: #5c5c5c;
```

Comment:
Would it be possible to pick an [already existing color](http://guardian.github.io/pasteup-palette/demo/)?

Reply:
Yeah, generally I prefer to use the existing vars for color... However, request to use #5c5c5c can from Ben. 🙇  

Labs needs to look visually distinct from Editorial content; I know that Labs related stuff have accrued so many colours, shades of grey being the most prolific. 

I am desperate to bring some unity (and deduping) to them, and ideally upgrade `labs` to an actual palette set like `feature` and `comment`...